src/controllers/dynakube/version/healthcheck.go

```diff
@@ -0,0 +1,79 @@
+package version
```

Comment:
Mmm, maybe we can move it to the dynakube/status package? as it sets the Dynakube OneAgent status, what do you think?
I would try to not create even more packages in the codebase

Reply:
IMO you mix 2 ideas: 
a) healthconfig related code
b) setStatusOneAgent 

So i would better create new package for healthconfig and use it in reconcile and move SetStatus to where it was.

Reply:
I tried to, but we end up with some import loops (because PullImageInfo is also in `version` package...)
I would say, let's leave it like this, it's not a big deal.